src/components/subscriptions/expiration/SubscriptionExpirationBanner.jsx

```diff
@@ -59,26 +59,26 @@ const SubscriptionExpirationBanner = ({ isSubscriptionPlanDetails }) => {
             <Alert.Heading>
               Your subscription contract is expiring soon
             </Alert.Heading>
-            Your current subscription contract will expire in {daysUntilExpiration} days.
+            Your current subscription contract will expire in {agreementNetDaysUntilExpiration} days.
             Renew your subscription today to minimize access disruption for your learners.
           </>
         )}
       </>
     );
   };
 
-  if (daysUntilExpiration > SUBSCRIPTION_DAYS_REMAINING_MODERATE) {
+  if (agreementNetDaysUntilExpiration > SUBSCRIPTION_DAYS_REMAINING_MODERATE) {
```

Comment:
The prop to use down here also depends on whether we're on the plan details page or the agreement "parent" page.  Maybe something like this toward the top of the component would be helpful:
{noformat}
const planDuration = isSubscriptionPlanDetails ? daysUntilExpiration : agreementNetDaysUntilExpiration;
{noformat}
And then you can use `planDuration` in all of the places we were previously using `daysUntilExpiration`.

Let me know if I'm completely off and missing something obvious here.

Reply:
This makes sense to me but maybe I'm unclear on the intended UX. Should we be passing messaging to our users that something is expiring/expired when they still have an on going subscription plan?

Example: 
User has 2 subscription plans, plan A (expired) and plan B (expires past the expiration warning threshold, +120 days)
When in the subscription plan detail view for plan A should we have messaging regarding its expiration?